ground/src/main/java/com/google/android/ground/ui/home/HomeScreenFragment.kt

```diff
@@ -78,20 +75,15 @@ class HomeScreenFragment :
   private lateinit var bottomSheetBehavior: BottomSheetBehavior<View>
   private lateinit var homeScreenViewModel: HomeScreenViewModel
   private lateinit var locationOfInterestSelectorViewModel: LocationOfInterestSelectorViewModel
-  private lateinit var mapContainerViewModel: HomeScreenMapContainerViewModel
-  private lateinit var surveySelectorViewModel: SurveySelectorViewModel
 
   private var progressDialog: ProgressDialog? = null
```

Comment:
We should also probably move the progress dialog into the survey selector screen and replace with a non-deprecated component. Fyi I'll be removing this in my upcoming cleanup PR removing shared repository state.

Reply:
I was hoping to resolve this separately since the scope of this PR is to replace the dialog with this new fragment. I've already filed a bug to create a progress dialog for survey selector fragment. So, please feel free to clean this in your upcoming PR.

Reply:
Sgtm!